src/nativeComponents/TableComponent/TableComponent.tsx

```diff
@@ -80,6 +83,14 @@ const getActionsColumn = ( actions: Array<TableRowAction>, t: TranslationFunctio
                         >
                             <Button
                                 title={t( ACTIONS )}
+                                primary={true}
+                                styles={{
+                                    root: {
+                                            backgroundColor: 'lightblue',
```

Comment:
move this to a class. multiple dimensions on the same line doesn't work on native. Can you ask Brandon to show you how to run the code on native?

Reply:
Moved the returned JSX here from getActionColumn call to its own functional component *TableActionsColumn* for clean up and to style button w/ class